### DISCUSSION TITLE
In-game chat

### DISCUSSION THREAD

**Discussion by CrsiX** [hopfenspace/runciv#2] [Ideas]
From our perspective, persistence is cheap, so why no persistent in-game chat as well? Additionally, this makes cross-device gaming where the devices aren't online all the time much more intuitive.

Additionally, this could be expanded to lobby-wide chatting, if there are lobbies (another discussion).

Additionally, user-to-user chats should be possible as well (not only user-to-all chat).

**Comment by NLECHOPPA106** [comment]
bruh

**Reply by myOmikron** [reply]
?

**Comment by romanovskaia21** [comment]
Chat between users is great for enhancing interactions between users.
